docs/content/porting-unreal-project-to-gdk.md

```diff
@@ -117,6 +117,14 @@ For more information on helper scripts, see [Helper scripts](https://github.com/
         Super::InitPlayerState();
     }
     ```
+    
+1. In the file(s) you added the `InitPlayerState` function definition, add the following includes:
```

Comment:
- In the file(s) you added ...-> In the file(s) where you added...

- includes -> `#include`s

(the "includes" one looks weird I know - we can decide not to do it like this if you think it's too strange-looking)

Reply:
(these look like things I missed first time round!)

Reply:
Adjusted.